docs/resources/dw_vw_impala.md

```diff
@@ -0,0 +1,44 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "cdp_dw_vw_impala Resource - terraform-provider-cdp"
+subcategory: ""
```

Comment:
When your generation and template is correct, the Data Warehouse string will be generated here